webui/npm-shrinkwrap.json

```diff
@@ -2055,36 +2055,11 @@
         "lodash": "^4.17.4"
       }
     },
-    "@nodelib/fs.scandir": {
```

Comment:
Can you revert the changes to this file?

Reply:
Sorry, my fault